test/marketplace/Abstracts.sol

```diff
@@ -2,17 +2,18 @@
 
 pragma solidity 0.8.4;
 
-abstract contract Erc20 {
-	function approve(address, uint256) virtual external returns (bool);
-	function transfer(address, uint256) virtual external returns (bool);
-	function balanceOf(address) virtual external returns (uint256);
-	function transferFrom(address, address, uint256) virtual public returns (bool);
+interface Erc20 {
```

Comment:
is this change to an interface because of the auto-generated getter for decimals? does it fail if you just put decimals in the abstract?

Reply:
Just gonna reference the ticket: https://github.com/code-423n4/2021-09-swivel-findings/issues/93

Reply:
i'm likely going to disregard this. 

we give 0 fucks about inherited contracts, the protocol is a closed loop.

Reply:
done #178